content/middlewares.md

```diff
@@ -44,6 +48,36 @@ export class LoggerMiddleware {
 }
 ```
 
+#### Use with Fastify
+
+The middleware with `fastify` "works", but it retrieves only the raw `req` and `res` objects and not the fastify wrapper. This is a side effect of how `middie` works and `fastify` already talks about it in its documents for more information see [here](https://www.fastify.io/docs/latest/Reference/Middleware/).
+
+> warning **Warning** The middleware in `fastify` has an IncomingMessage request type `FastifyRequest['raw']` and a ServerResponse type `FastifyReply['raw']`.
+
+```typescript
+@@filename(logger.middleware)
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { FastifyRequest, FastifyReply } from 'fastify';
+
+@Injectable()
+export class LoggerMiddleware implements NestMiddleware {
+  use(req: FastifyRequest, res: FastifyReply, next: () => void) {
```

Comment:
You should fix the types here. You mention earlier that the types _should_ be `FastifyRequest['raw']` and `FastifyReply['raw']` so you should use those here as well

Reply:
Right!👍, I forgot to update it when I added in warning above 😅, however I have now updated the example. (see [here](https://github.com/nestjs/docs.nestjs.com/pull/2589/commits/cdda539514ba622acecc4b9dcf88f8d13e935c62)) ✅

Thanks for the review 🙌